lib/test.js

```diff
@@ -104,51 +112,52 @@ Test.prototype.plan = function (count) {
 };
 
 Test.prototype.run = function () {
-	this.promise = {};
+	var self = this;
+
+	this.promise = Promise.pending();
```

Comment:
`Promise.pending()`? Couldn't find this anywhere in the Bluebird docs: http://bluebirdjs.com/docs/api-reference.html